src/main/java/uk/gov/hmcts/cmc/claimstore/services/ccd/CallbackHandlerFactory.java

```diff
@@ -24,9 +27,12 @@
     @Autowired
     public CallbackHandlerFactory(
         MoreTimeRequestedCallbackHandler moreTimeRequestedCallbackHandler,
-        Optional<GenerateOrderCallbackHandler> generateOrderCallbackHandler) {
+        Optional<GenerateOrderCallbackHandler> generateOrderCallbackHandler,
```

Comment:
We should avoid passing Optional in parameters.

Reply:
👍 will address as part of the base PR

Reply:
Done! changed it in https://github.com/hmcts/cmc-claim-store/pull/893. Please ignore this in this PR :D

Reply:
🤹‍♀